testing/thor/LICENSE-APACHE

```diff
@@ -0,0 +1,201 @@
+                              Apache License
```

Comment:
Hmm, this subfolder now claims to have an Apache/MIT license, but jormungandr as a whole has no declared license. We should give some care about this.